e2e/packages/sync-test/package.json

```diff
@@ -12,7 +12,7 @@
     "@latticexyz/schema-type": "link:../../../packages/schema-type",
     "@latticexyz/store": "link:../../../packages/store",
     "@latticexyz/utils": "link:../../../packages/utils",
-    "@playwright/test": "^1.35.0",
+    "@playwright/test": "^1.36.0",
```

Comment:
might be helpful to use `1.36.0` here instead of `^1.36.0` so we get the exact version rather than potential patch version bumps